<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>连接配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit
{
	background: #FFFFFF;
	border: 1px solid #DADFE3;
	border-radius: 4px;
	padding-left:12px;
	font-family: PingFang SC;
	font-style: normal;
	font-weight: normal;
	font-size: 14px;
	line-height: 22px;
}

QWidget#LoginForm
{
	background: #FFFFFF;
	border-radius: 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{
font-family: PingFang SC;
font-style: normal;
font-weight: 600;
font-size: 28px;
line-height: 36px;
display: flex;
align-items: center;
color: #333333;
}</string>
        </property>
        <property name="text">
         <string>MQTT 配置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="brokerUrlLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>MQTT Broker 地址，如 tcp://localhost:1883</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="agentIdLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>智能体 ID (Agent ID)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="clientIdLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>客户端 ID (Client ID)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startVoiceChatBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: #007AFF;
	border-radius: 4px;
	color:#FFFFFF;
	font-family: PingFang SC;
	font-style: normal;
	font-weight: 600;
	font-size: 18px;
	line-height: 22px;
}</string>
     </property>
     <property name="text">
      <string>开始语音通话</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
